converter.py

```diff
@@ -565,6 +571,9 @@ def _bytes_to_snowflake(self, value):
     def _bool_to_snowflake(self, value):
         return value
 
+    def _bool__to_snowflake(self, value): #test
+        return value
```

Comment:
I think this should be `return bool(value)` to turn it into a `bool` instead of a `np.bool_`